install-unicorn-macos.sh

```diff
@@ -0,0 +1,20 @@
+# This script installs the unicorn engine on MacOS.
+# Check if the OS is MacOS
+if [ "$(uname)" == "Darwin" ]; then
+
+    # Clone everything
+    rm -rf ../unicorn
```

Comment:
this can be dangerous... the user may create a folder named `unicorn` and you remove it without telling the user. Check the folder and if exists, exit and telling the user to rename, or add an optional argument for this script.